pre_commit/languages/r.py

```diff
@@ -60,10 +61,11 @@ def _prefix_if_non_local_file_entry(
 
 def _rscript_exec() -> str:
     r_home = os.environ.get('R_HOME')
+    program = win_exe('Rscript')
     if r_home is None:
-        return 'Rscript'
+        return program
```

Comment:
this one doesn't actually need to change since the PATH lookup will utilized the same extension lookup windows does -- it's only when it's "explicitly" passed a path that it skips that normalization

Reply:
Right...